OBSERVABILITY.md

```diff
@@ -0,0 +1,93 @@
+## Observability with OpenTelemetry
+
+This Cloud Spanner client supports [OpenTelemetry Traces](https://opentelemetry.io/), which gives insight into the client internals and aids in debugging/troubleshooting production issues.
+
+By default, the functionality is disabled. You shall need to add OpenTelemetry dependencies, and must configure and
+enable OpenTelemetry with appropriate exporters at the startup of your application:
+
+**Table of contents:**
+
+* [Observability](#observability)
+  * [Tracing](#tracing)
+    * [OpenTelemetry Dependencies](#opentelemetry-dependencies)
+    * [OpenTelemetry Configuration](#opentelemetry-configuration)
+    * [SQL Statement span annotation](#sql-statement-span-annotation)
+    * [OpenTelemetry gRCP instrumentation](#opentelemetry-grpc-instrumentation)
+    * [Tracing Sample](#tracing-sample)
+
+### Tracing
+
+#### OpenTelemetry Dependencies
+
+Use a trace exporter of your choice, such as Google Cloud Trace like below:
+
+```javascript
+const {
+  TraceExporter,
+} = require('@google-cloud/opentelemetry-cloud-trace-exporter');
+const exporter = new TraceExporter();
+```
```

Comment:
```suggestion
Add the following dependencies in your `package.json` or install them directly.

```json
// Required packages for OpenTelemetry SDKs
"@opentelemetry/sdk-trace-base": "^1.26.0",
"@opentelemetry/sdk-trace-node": "^1.26.0",

// Package to use Google Cloud Trace exporter
"@google-cloud/opentelemetry-cloud-trace-exporter": "^2.4.1"

// Packages to enable gRPC instrumentation
"@opentelemetry/instrumentation": "^0.53.0",
"@opentelemetry/instrumentation-grpc": "^0.53.0",
```
```

Reply:
Done, thank you @surbhigarg92!